### PR TITLE
[skip ci] Bump infra scripts to 6.5.0

### PR DESCRIPTION
**Pull request by caglar10ur**
** DO NOT MERGE FOR NOW **

### CODE REVIEW

**Comment by hmahmood** [comment]
lgtm for when you _do_ want to merge :-)

**Comment by fdawg4l** [comment]
lgtm

**Comment by caglar10ur** [comment]
marking it as WIP, realized that 6.5 comes with goodies (http://www.virtuallyghetto.com/2016/10/nested-esxi-enhancements-in-vsphere-6-5.html) and some of them needs to be implemented on govmomi first (eg; nvme support)

File: infra/machines/vcsa/create-esxi-vm.sh [hunk 1]
```diff
@@ -76,6 +76,7 @@ username=$GOVC_USERNAME
 password=$GOVC_PASSWORD
 unset GOVC_USERNAME GOVC_PASSWORD
 
+
```
Comment (dougm) [inline infra/machines/vcsa/create-esxi-vm.sh]: maybe undo this extra blank line when you rebase :)

File: infra/machines/vcsa/create-esxi-vm.sh [hunk 2]
```diff
@@ -109,22 +110,22 @@ if [ "$(govc env -x GOVC_URL_HOST)" = "." ] ; then
 
     cat > "$vmx" <<EOF
 config.version = "8"
-virtualHW.version = "11"
+virtualHW.version = "13"
```
Comment (dougm) [inline infra/machines/vcsa/create-esxi-vm.sh]: These changes work ok with Fusion 8.x?  I'm still on 7.x and the current set of properties works with esx 6.5.. but suppose I should upgrade at somepoint.

**Comment by dougm** [comment]
lgtm

File: infra/machines/vcsa/create-esxi-vm.sh (hunk 2 above)
Comment (caglar10ur) [inline infra/machines/vcsa/create-esxi-vm.sh]: Haven't tried them on fusion or workstation 🤒

**Comment by mdubya66** [comment]
Approved for merge

File: infra/machines/vcsa/create-esxi-vm.sh (hunk 2 above)
Comment (caglar10ur) [inline infra/machines/vcsa/create-esxi-vm.sh]: Looks like fusion does not support new HW version so dropped fusion changes

**Comment by caglar10ur** [comment]
merging it as the code already reviewed

**Pull request merged**